windows-02.md

```diff
@@ -31,63 +27,61 @@ npm install -g degit
+Credentials saved to file: [%APPDATA%\gcloud\application_default_credentials.json]
 
-- Click on the link noted above
-- You'll be taken to a Service Account Details page, where there are some tabs. Click on the one called **Keys**.
+These credentials will be used by any library that requests Application Default Credentials (ADC).
 
-![Create key](images/createkey.png)
+Quota project "[RECENTLY_CREATED_PROJECT_ID]" was added to ADC which can be used by Google client libraries for billing and quota. Note that some services may still bill the project owning the resource.
+% 
+```
 
-- Use the **ADD KEY** button then **Create new key** option, choose the **JSON** option and click **CREATE**.
-- A file will be saved on your machine. _This file is important_ and you need to keep it on your machine! I renamed my file `google_drive_fetch_token.json` and put it in same folder with all of my other icj class projects, for example: `/c/Users/christian/Documents/icj/google_drive_fetch_token.json`.
-- Next go the [API Library page](https://console.cloud.google.com/apis/library).
-- Use the search to find the **Google Docs API** and select it.
-  - Make sure `icj-project` is selected in the top nav near the Google Cloud Services logo.
-  - Then click on the **Enable** buttton to activate it.
-- Use the search bar to find **Google Sheets API** and choose it and **Enable** it.
+We'll test this with the icj-project-template when the time comes. If you use OneDrive, you might have to use **Git Bash** for some steps instead of the terminal within VS Code.
 
 ### Setting up the environment variable
-
 > There _may_ be some issues setting this up on Windows machines that use OneDrive. It has been successfully installed on a non-OneDrive setup using this method.
 
 We are setting this environment variable to authenticate ourselves to Google using the information in the json file you just created.
 
-- Open a new Git Bash prompt
-- Do `code .bash_profile` to open your ".bash_profile" file in VS Code. You should see some stuff there already from other configurations. (Hollar if you don't as that means you are likely in the wrong file.)
-- Add the text below to this file, but **with your home directory name as `your_username`**.
-
-```bash
-# Google Auth
-export GOOGLE_APPLICATION_CREDENTIALS="C:/Users/your_username/Documents/icj/google_drive_fetch_token.json"
-```
-
-If you have and use Microsoft OneDrive, then you might need to modify that path just a bit:
-
+1. Open a new Git Bash prompt.
+2. Enter `code .bash_profile` to open your `.bash_profile` file in VS Code. You should see some stuff there already from other configurations. (Holler if you don't as that means you are likely in the wrong file.)
 ```bash
 # Google Auth
-set GOOGLE_APPLICATION_CREDENTIALS="C:/Users/your_username/OneDrive/Documents/icj/google_drive_fetch_token.json"
+set GOOGLE_APPLICATION_CREDENTIALS="$APPDATA\gcloud\application_default_credentials.json"
```

Comment:
> Once you login to Google Cloud, your credentials will be stored in a JSON file, under the following default locations:
> Windows: %APPDATA%\gcloud\application_default_credentials.json

[Link](https://towardsdatascience.com/google-application-credentials-python-ace518208a7#:~:text=Once%20you%20login%20to%20Google,config%2Fgcloud%2Fapplication_default_credentials.)

Reason it is `$APPDATA` instead of `%APPDATA%`: [Link](https://superuser.com/questions/1284276/how-to-use-localappdata-with-git-bash-in-windows)